source/include/protocol/cs_TrackableParserPackets.h

```diff
@@ -34,22 +34,13 @@ struct __attribute__((__packed__)) trackable_parser_cmd_get_filer_summaries_t {
 
 // ------------------ Return values ------------------
 
+struct __attribute__((__packed__)) trackable_parser_cmd_upload_filter_ret_t {};
 
-struct __attribute__((__packed__)) trackable_parser_cmd_upload_filter_ret_t {
+struct __attribute__((__packed__)) trackable_parser_cmd_remove_filter_ret_t {};
 
-};
-
-struct __attribute__((__packed__)) trackable_parser_cmd_remove_filter_ret_t {
-
-};
-
-struct __attribute__((__packed__)) trackable_parser_cmd_commit_filter_changes_ret_t {
+struct __attribute__((__packed__)) trackable_parser_cmd_commit_filter_changes_ret_t {};
 
-};
-
-struct __attribute__((__packed__)) trackable_parser_cmd_get_filer_summaries_ret_t {
-
-};
+struct __attribute__((__packed__)) trackable_parser_cmd_get_filer_summaries_ret_t {};
```

Comment:
Why even define those?

Reply:
Had to be defined at some point. Sneaked into this branch because I started working on the trackable parser before the cuckoo filter packets were finished.